Userland/Libraries/LibGfx/ICCProfile.h

```diff
@@ -45,17 +45,49 @@ enum class DeviceClass : u32 {
 };
 char const* device_class_name(DeviceClass);
 
+// ICC v4, 7.2.6 Data colour space field, Table 19 — Data colour space signatures
+enum class ColorSpace : u32 {
+    nCIEXYZ = 0x58595A20,       // 'XYZ '
+    CIELAB = 0x4C616220,        // 'Lab '
+    CIELUV = 0x4C757620,        // 'Luv '
+    YCbCr = 0x59436272,         // 'YCbr'
+    CIEYxy = 0x59787920,        // 'Yxy '
+    RGB = 0x52474220,           // 'RGB '
+    Gray = 0x47524159,          // 'GRAY'
+    HSV = 0x48535620,           // 'HSV '
+    HLS = 0x484C5320,           // 'HLS '
+    CMYK = 0x434D594B,          // 'CMYK'
+    CMY = 0x434D5920,           // 'CMY '
+    TwoColor = 0x32434C52,      // '2CLR'
+    ThreeColor = 0x33434C52,    // '3CLR'
+    FourColor = 0x34434C52,     // '4CLR'
+    FiveColor = 0x35434C52,     // '5CLR'
+    SixColor = 0x36434C52,      // '6CLR'
+    SevenColor = 0x37434C52,    // '7CLR'
+    EightColor = 0x38434C52,    // '8CLR'
+    NineColor = 0x39434C52,     // '9CLR'
+    TenColor = 0x41434C52,      // 'ACLR'
+    ElevenColor = 0x42434C52,   // 'BCLR'
+    TwelveColor = 0x43434C52,   // 'CCLR'
+    ThirteenColor = 0x44434C52, // 'DCLR'
+    FourteenColor = 0x45434C52, // 'ECLR'
+    FifteenColor = 0x46434C52,  // 'FCLR'
+};
+char const* color_space_name(ColorSpace);
```

Comment:
Can we make this `StringView`?

Reply:
Why? It returns a string literal, isn't a const char the fastest, most binary size effective way to handle those?

Reply:
As in: using a `""sv` has the effect of computing the string's length at compile time and storing that in the binary. That makes sense if the function gets called in many places that want the length, but that's (at least currently) not the case here. (Likely it'll never be the case -- we don't do any computation on the result of this function.)

Reply:
I mentioned it because it's *only* used in a context that wants the length (currently at least, dunno what the plan for the future is for method). If it's just forever a debugging thing, then comme ci comme ça :man_shrugging:

Reply:
I want to use it in `icc`, and once the class does parse color curves I'm thinking about adding a GUI ICC viewer that'd also display it as a string somewhere. But in both cases it's not hot code, it's a short string, and computing the length on demand seems marginally nicer to me than making LibGfx a few bytes larger. (Granted, it doesn't matter a ton either way.)

Reply:
:+1: fair enough